doc/source/examples/incompressible-flow/2d-lid‐driven-cavity-flow/lid‐driven-cavity-flow.rst

```diff
@@ -0,0 +1,306 @@
+
+.. math::
+
+  \Delta t_{n+1} = \alpha \Delta t_{n}
+
+
+where :math:`\alpha` is the ``adaptative time step scaling`` .
+
+A complimentary strategy is to also start the problem from a closer initial condition. This is achieved with the ``initial conditions`` subsection:
+
+.. code-block:: text
+
+  subsection initial conditions
+    set type      = viscous
+    # viscosity for viscous initial conditions
+    set viscosity = 0.1
+  end
+
+Here we use ``viscous`` initial condition in which an auxiliary steady-state problem at a higher viscosity is used to generate a more realistic initial condition to the problem at hand.
+
+Results
+~~~~~~~~~
+
+We first carry out the simulations with an initial refinement 7. Using Paraview, the steady-state velocity profile and the streamlines can be visualized:
+
+.. image:: images/result_re_7500.png
+    :alt: velocity distribution
+    :align: center
+
+We see that new recirculation regions occur in the flow.
+
+Using a similar python script, we can compare the results obtained with those from the literature. We note that there is quite a good agreement with the results from the literature, even in the top region where the velocity field changes abruptly.
+
+.. image:: images/re_7500_mesh_7/lethe_ghia_re_7500_comparison.png
+    :alt: re_7500_comparison_mesh_7
+    :align: center
+
+.. image:: images/re_7500_mesh_7/lethe_ghia_re_7500_comparison_zoom.png
+    :alt: re_7500_comparison_mesh_7
+    :align: center
+
+Increasing the number of cells by a factor 4 (to ~65k) allows for an even slightly better agreement.
+
+.. image:: images/re_7500_mesh_8/lethe_ghia_re_7500_comparison.png
+    :alt: re_7500_comparison_mesh_8
+    :align: center
+
+.. image:: images/re_7500_mesh_8/lethe_ghia_re_7500_comparison_zoom.png
+    :alt: re_7500_comparison_mesh_8
+    :align: center
+
+
+Possibilities for extension
+----------------------------
+
+- **Validate at higher Reynolds number:** The Erturk 2005 data within the example investigates this case up to a Reynolds number of 20000. Using the adjoint time-stepping ``steady_bdf`` scheme, it is an interesting exercise to simulate these more complex cases.
+- **High-order methods:** Lethe supports higher order interpolation. This can yield to much better results with an equal number of degrees of freedom than traditional second-order (Q1-Q1) methods, especially at higher Reynolds number. 
```

Comment:
```suggestion
- **High-order methods:** Lethe supports higher order interpolation. This can yield much better results with an equal number of degrees of freedom than traditional second-order (Q1-Q1) methods, especially at higher Reynolds numbers. 
```